packages/table/src/table.tsx

```diff
@@ -452,8 +457,8 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
 
         return !Utils.shallowCompareKeys(this.props, nextProps, propKeysBlacklist)
             || !Utils.shallowCompareKeys(this.state, nextState, stateKeysBlacklist)
-            || !Utils.deepCompareKeys(this.props, nextProps, ["selectedRegions"])
-            || !Utils.deepCompareKeys(this.state, nextState, ["selectedRegions"]);
+            || !Utils.deepCompareKeys(this.props, nextProps, Table.SHALLOW_COMPARE_PROP_KEYS_BLACKLIST)
```

Comment:
These lists happen to be complementary, so I'm using the shallow-compare blacklists for the deep-compare whitelists. Is that too clever? Should I create separate `Table.DEEP_COMPARE_{PROP,STATE}_KEYS_WHITELIST`s for the sake of robustness? (I could still set those to equal the shallow-compare blacklist arrays, with a comment to clarify why that's okay.)

Reply:
I'm not sure what you mean by this, so... um, possibly a comment anyways? What are you doing that's odd, exactly?

Reply:
Eh I think I was overthinking it. Fine as is IMO.